test/unit/vdom/skip.js

```diff
@@ -0,0 +1,60 @@
+import { define, prop, props, symbols, vdom } from '../../../src/index';
+import afterMutations from '../../lib/after-mutations';
+import fixture from '../../lib/fixture';
+
+describe('vdom/skip', () => {
+  it('should skip the element children', done => {
+    const Elem = define('x-test', {
+      props: {
+        num: prop.number()
+      },
+      render() {
+        vdom.element('div', { skip: true }, () => {
+          vdom.element('span', () => {
```

Comment:
What's the rationale for this *not* throwing an exception? (since skip was declared on the parent). I thought the idom behaviour was for this to throw?